src/wp-includes/option.php

```diff
@@ -1847,15 +1847,26 @@ function register_initial_settings() {
 		) );
 	}
 
-	register_setting( 'general', 'login_custom_logo', array(
+	register_setting( 'general', 'login_custom_logo_check', array(
 		'show_in_rest' => array(
-			'name' => 'login_custom_logo',
+			'name' => 'login_custom_logo_check',
 		),
 		'type'         => 'boolean',
 		'default'      => false,
 		'description'  => __( 'Use the site logo as the login image' ),
 	) );
 
+	register_setting( 'general', 'login_custom_logo', array(
+		'show_in_rest' => array(
+			'name' => 'login_custom_logo',
+			'schema'  => array(
+				'format' => 'uri',
```

Comment:
This should probably be an integer (attachment ID) instead of a URI?

Reply:
Thought about it some more and yes, this needs to be updated to store an attachment ID. If an image URI is stored, then this feature will break when a site is moved to a different domain for example.